src/Core/Silk.NET.SilkTouch/MarshalBuilder.cs

```diff
@@ -0,0 +1,52 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Collections.Generic;
+
+namespace Silk.NET.SilkTouch
+{
+    public class MarshalBuilder
+    {
+        private readonly List<Middleware> _middlewares = new List<Middleware>();
```

Comment:
Technically middleware is the plural of middleware (like sheep and sheep) but I'm not gonna make you change that because that'd be a waste of time just to satisfy my pernickety brain.